raw_install_python.yml

```diff
@@ -50,8 +50,25 @@
       when: stat_apt.rc == 0
 
     - name: install python for opensuse
-      raw: zypper -n install python-base
+      raw: zypper -n install python-base python-xml
       register: result
       until: result is succeeded
       when: stat_zypper.rc == 0
-  when: not True in (systempython.results | selectattr('stat', 'defined') | map(attribute='stat.exists') | list | unique)
\ No newline at end of file
+  when: not True in (systempython.results | selectattr('stat', 'defined') | map(attribute='stat.exists') | list | unique)
+
+- name: check for python2 on SUSE/openSUSE
```

Comment:
this task is a duplicate of the first one

Reply:
Yes, but I was not sure if I could get only the detection of `/usr/bin/python` out of the first task.